hack/run-functests.sh

```diff
@@ -7,7 +7,7 @@ which ginkgo
 if [ $? -ne 0 ]; then
 	echo "Downloading ginkgo tool"
 	#go install github.com/onsi/ginkgo/ginkgo
-  go install github.com/onsi/ginkgo/ginkgo@v1.16.5
+  GOFLAGS='' go install github.com/onsi/ginkgo/ginkgo@v1.16.5
```

Comment:
could you please add a comment/note in the commit message about what was the problem and why this fix resolves the problem?

Reply:
Sure will add the comment.